samples/cpp/whisper_speech_recognition/README.md

```diff
@@ -17,24 +17,21 @@ optimum-cli export openvino --trust-remote-code --model openai/whisper-base whis
 
 Prepare audio file in wav format with sampling rate 16k Hz.
 
-## Run
+```sh
+wget https://storage.openvinotoolkit.org/models_contrib/speech/2021.2/librispeech_s5/how_are_you_doing_today.wav
```

Comment:
You can just post a link. It's up to a reader how to download it.

Reply:
done